android/java/org/chromium/chrome/browser/app/domain/NetworkModel.java

```diff
@@ -65,6 +64,7 @@ public class NetworkModel implements JsonRpcServiceObserver {
     private final MediatorLiveData<List<NetworkInfo>> _mPrimaryNetworks;
     private final MediatorLiveData<List<NetworkInfo>> _mSecondaryNetworks;
     private Map<String, NetworkSelectorModel> mNetworkSelectorMap;
+    private MutableLiveData<NetworkLists> _mNetworkLists;
```

Comment:
It can be `final`.

Reply:
Updated https://github.com/brave/brave-core/pull/19467#discussion_r1278958347